0098-mobile-subdao-quality-of-service-requirements.md

```diff
@@ -145,6 +146,8 @@ In the example above, the average would be 0.6875, and wojld be utilized during
 ## Rationale 
 As the Helium 5G network matures, it’s vitally important that the quality of the network and deployments provide usable and consistent coverage. 
 
+## Deployment Impact
+Nova Labs will have to change the speedtest requirements to incorporate the new "Acceptable" multiplier of 0.75X upon passing. 
```

Comment:
```suggestion
Protocol engineers will have to change the speedtest requirements to incorporate the new "Acceptable" multiplier of 0.75X upon passing. 
```

Reply:
Small clarification, NL could be the ones to do this but any one can open a PR to implement this change in https://github.com/helium/oracles/blob/main/mobile_verifier/src/reward_shares.rs

Reply:
Is there a way for me to “accept” the accepted change, or can you push it?